moci/js/modules/system.js

```
@@ -100,9 +94,7 @@ export default class SystemModule {
 			this.subTabs.cleanup();
 			this.subTabs = null;
 		}
-		this.cleanups.filter(Boolean).forEach(fn => {
-			fn();
-		});
+		this.cleanups.filter(Boolean).forEach(fn => fn());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)